util/checkers/proposal.go

```diff
@@ -6,16 +6,27 @@ import (
 	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	gov1b1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
 )
 
+var govModuleAddr = authtypes.NewModuleAddress(govtypes.ModuleName).String()
```

Comment:
This fails. Super strange. If I move this function call into `ValidateProposal` then there is no panic.  No idea why it fails.